APP_DEVELOPMENT.md

```diff
@@ -160,6 +174,31 @@ These are custom components we've developed, in addition to NativeBase readymade
 
 Displays the wide version of the STA logo
 
+#### STA Multi-coloured Matchstick Logo
+
+![STA Matchstick](app/src/Assets/Images/STA%20Matchstick%20Logo.png)
+
+STA Matchstick Logo with seven, rotated multi-coloured matchsticks
+
+#### STA Single Matchstick
+
```

Comment:
Great additions David! Well done on adding three reusable components, awesome :) 

It looks like the naming convention here is to add the reference to the components here with the name as it is in the code, i.e. StaRibbon, Checkbox, ProgressBar.